Update direct_link_generator.py

jusidama18 opened the pull request.

anasty17 commented:
Bro i guess it's working but I have added header check .. the link didn't pass in generator even ... i have fixed this .. but didn't pusb yet .... unless if there is another issue

jusidama18 commented:
> Bro i guess it's working but I have added header check .. the link didn't pass in generator even ... i have fixed this .. but didn't push yet .... unless if there is another issue

okay, will close for now
